aws/resource_aws_apigatewayv2_api.go

```diff
@@ -177,6 +216,11 @@ func resourceAwsApiGatewayV2ApiCreate(d *schema.ResourceData, meta interface{})
 
 	d.SetId(aws.StringValue(resp.ApiId))
 
+	err = resourceAwsAPIGatewayV2ImportOpenAPI(d, meta)
+	if err != nil {
+		return fmt.Errorf("error update API Gateway v2 API (%s) with OpenAPI spec: %s", d.Id(), err)
```

Comment:
```suggestion
		return err
```

The relevant error information was captured in `resourceAwsAPIGatewayV2ImportOpenAPI` so there's no need to wrap here.